lib/load.php

```diff
@@ -104,6 +102,9 @@ function gutenberg_is_experiment_enabled( $name ) {
 
 // WordPress 6.2 compat.
 require __DIR__ . '/compat/wordpress-6.2/script-loader.php';
+require __DIR__ . '/compat/wordpress-6.2/get-global-styles-and-settings.php';
```

Comment:
In https://github.com/WordPress/gutenberg/pull/45168 I missed the place where these imports should be located.